frontend/src/index.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from 'react-dom';
 import App from './scenes/App/App';
-import './styles/index.scss';
+import {BrowserRouter as Router, Route} from "react-router-dom";
 
-ReactDOM.render(<App />, document.getElementById('root'));
+render(
+    <Router>
+        <Route path="/"><App /></Route>
+    </Router>,
```

Comment:
Now you wrapped the App component into the Router, but we need to do it vice versa. Our index.tsx file should stay the same - `render(<App />, document.getElementById('root'));`. And then App component will look like this: 
`<Router><Routing /></Router>`. The Routing component should be placed in containers and will contain all routes. 
Also could you please move the App component folder from scenes to containers? And wrap the Routing file into react-redux Provider (I noticed that forgot to do that). Thanks!